src/domain/models/CountryCode.ts

```diff
@@ -3,6 +3,7 @@
  * @enum {string}
  */
 export enum CountryCode {
+  Germany = "DE",
```

Comment:
It would be better not to add this line until the first challenge has been published.

It would not break the app, but users would be able to navigate there and see 1 upcoming challenge in May.

Reply:
Reverted this part